README.md

```diff
@@ -36,6 +36,32 @@ It will install WireGuard (kernel module and tools) on the server, configure it,
 
 Run the script again to add or remove clients!
 
+## Configure clients
+
+### RHEL 8
+
+* Install the necessary packages:
+
+```
+sudo dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
+sudo dnf install dnf-plugins-core
+sudo dnf config-manager --enable elrepo
+sudo dnf install kmod-wireguard wireguard-tools
+```
+
+* Create file `/etc/wireguard/wg0.conf` with contents of the client config generated by the script (copy paste from server).
```

Comment:
```suggestion
* Create file in `/etc/wireguard/` with the name you want the network interface to be with extension of `.conf`. IE: `/etc/wireguard/wg0.conf`, then put in the contents of the client config generated by the script (copy paste from server). Also run `sudo chown -R root. /etc/wireguard ; chmod -R 600 /etc/wireguard` for proper permissions.

To enable the connection run `sudo wg-quick up [Interface name]`, like `sudo wg-quick up wg0` and to disable `wg-quick down [Interface name]`
```
This allows for multiple WireGuard VPN's and user customization. And how they can enable and disable